antiope-sqs/src/Antiope/SQS.hs

```diff
@@ -60,13 +61,15 @@ ackMessages (QueueUrl queueUrl) msgs = do
         _  -> return $ Left DeleteMessageBatchError
     else return $ Left DeleteMessageBatchError
 
+msgInBody :: Text -> Maybe Text
```

Comment:
`messageBody`?
Also, do we need functions  operating  in `Text`? Why not on `Message`?

Reply:
`s3Location'` is useful (the Text one) because we have these in a database and we want to do things with them.